plugins/jenkins/src/api/JenkinsApi.ts

```diff
@@ -19,7 +19,9 @@ import {
   DiscoveryApi,
   IdentityApi,
 } from '@backstage/core-plugin-api';
-import { EntityName, EntityRef } from '@backstage/catalog-model';
+import type { EntityName, EntityRef } from '@backstage/catalog-model';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
do you need this?

Reply:
Otherwise I'm getting an error if I run backstage lint, I can try to add this dependency, if it is ok for you :)